src/Categories/Morphism/Properties.agda

```diff
@@ -150,3 +150,66 @@ _∘↠_ : B ↠ C → A ↠ B → A ↠ C
 f ∘↠ g = record { mor = mor f ∘ mor g ; epi = Epi-∘ (epi f) (epi g) }
   where
     open _↠_
+
+--------------------------------------------------------------------------------
+-- Isomorphism from a section and a retraction
+
+EpicRetract⇒Iso : ∀ {X Y} {f : Y ⇒ X} {r : X ⇒ Y} → 
+                   r RetractOf f → Epi f → Iso f r
+EpicRetract⇒Iso {X} {Y} {f} {r} rf≈id e = record { 
+  isoˡ = rf≈id ; 
+  isoʳ = e (f ∘ r) id (begin
```

Comment:
Please use a local `where` clause instead of an inline proof - that scales better in the long term (i.e. for the users of the resulting `Iso`)

Reply:
Unfortunately, I am still new to agda, and I am not sure how to use a where clause in this situation. What change are you suggesting?